root/components/home/Explore.js

```diff
@@ -0,0 +1,186 @@
+                </a>
+              </h5>
+              <p className="card-text">
+                {l(`In 2000, Gracenote took over the free
+                  CDDB project and commercialized it,
+                  essentially charging users for accessing
+                  the very data they themselves contributed.
+                  In response, Robert Kaye founded MusicBrainz.
+                  The project has since grown rapidly from
+                  a one-man operation to an international
+                  community of enthusiasts that appreciates
+                  both music and music metadata. Along the way,
+                  the scope of the project has expanded from
+                  its origins as a mere CDDB replacement
+                  to the true music encyclopedia MusicBrainz is today.`)}
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div className="col-md-6 d-flex">
+          <div className="card">
+            <div className="card-body">
+              <h5 className="card-title">
+                <a
+                  href="https://musicbrainz.org/doc/MusicBrainz_Database"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('MusicBrainz Database')}
+                </a>
+              </h5>
+              <p className="card-text">
+                {l(`The MusicBrainz Database stores all of the
+                  various pieces of information we collect
+                  about music, from artists and their
+                  releases to works and their composers,
+                  and much more. Most of the data in the
+                  MusicBrainz Database is licensed under CC0,
+                  which effectively places the data
+                  into the Public Domain. The remaining data is
+                  released under the Creative Commons
+                  Attribution-NonCommercial-ShareAlike 3.0 license.
+                  All our data is available for commercial licensing.
+                  {l('If you are interested in licensing this data
+                  {l('for commercial use, please contact us.`)}
+              </p>
+            </div>
+          </div>
+        </div>
+        <div className="col-md-6 d-flex">
+          <div className="card">
+            <div className="card-body">
+              <h5 className="card-title">
+                <a
+                  href="https://musicbrainz.org/doc/Editing_FAQ"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Varied and Never Ending')}
+                </a>
+              </h5>
+              <p className="card-text">
+                {l(`As an encyclopedia and as a community,
+                  MusicBrainz exists only to collect as much
+                  information about music as we can. We do not
+                  discriminate or prefer one &quot;type&quot; of
```

Comment:
We generally indent multiline strings so that they are all on the same line as the first word of the first line (so just under "As" here)

Reply:
Right